.babelrc

```diff
@@ -1,6 +1,8 @@
 {
   "presets": ["es2015", "stage-1", "react"],
   "plugins": [
+    "flow-react-proptypes",
+    "transform-flow-strip-types",
```

Comment:
You sure? I see `require("babel-plugin-transform-flow-strip-types")` on [Line #4](https://github.com/babel/babel/blob/master/packages/babel-preset-react/index.js#L4)